Scripts/Utilities/MathUtilsExt.cs

```diff
@@ -204,6 +204,16 @@ public static void LerpTransform(Transform source, Vector3 targetPosition, Quate
 			source.position = Vector3.Lerp(source.position, targetPosition, t);
 			source.rotation = Quaternion.Slerp(source.rotation, targetRotation, t);
 		}
+
+		/// <summary>
+		/// Perform a smooth in+out interpolation of a 0-1 lerp value
+		/// </summary>
+		/// <param name="lerpAmount">The 0-1 lerp value that is to be shaped by this function</param>
+		/// <returns>The 0-1 smoothed lerp value</returns>
+		public static float SmoothInOutLerpFloat(float lerpAmount)
+		{
+			return Mathf.Pow(lerpAmount, 3) * (lerpAmount * (6f * lerpAmount - 15f) + 10f);
```

Comment:
Would be good to link to an article about this or explain why this particular polynomial

Reply:
This is just a personal formula/preference I've shaped over time; there isn't an article to back it up.  It just "feels" good in certain UI/anim aesthetics.

Reply:
// This polynomial feels good

But seriously, some explanation would be great. Maybe this? https://www.wolframalpha.com/input/?i=t%5E3+*+t+*+(6+*+t+-+15)+%2B+10)

Also, wolfram seems to think this can be simplified ;)

Reply:
Technically, this would be the proper wolfram translation https://www.wolframalpha.com/input/?i=t%5E3+*+(t+*+(6+*+t+-+15)+%2B+10)